open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/manage/validator/VRPConsentRequestValidator.java

```diff
@@ -0,0 +1,736 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
license year for the new files should be 2024. can you please check all newly added files

Reply:
Fixed in [876c68f]